commit 4751111ab26017aeb0f9f74ef7762087b12331f2
Author: John Levon <john.levon@joyent.com>
Date:   2019-08-22T10:12:02+00:00 (7 weeks ago)
    
    OS-7961 'debug' loader option is a little obscure
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Rob Johnston <rob.johnston@joyent.com>
    Approved by: Rob Johnston <rob.johnston@joyent.com>

diff --git a/boot/forth/menu.rc b/boot/forth/menu.rc
index bb9a2375..9fa50c90 100644
--- a/boot/forth/menu.rc
+++ b/boot/forth/menu.rc
@@ -127,20 +127,20 @@ set optionsansi_caption[3]="^[1mV^[merbose......... ^[34;1mOff^[m"
 set optionstoggled_ansi[3]="^[1mV^[merbose......... ^[32;7mOn^[m"
 
 set optionsmenu_init[4]="init_kmdb"
-set optionsmenu_caption[4]="k[m]db................ Off"
-set optionstoggled_text[4]="k[m]db................ On"
+set optionsmenu_caption[4]="Load [k]mdb........... Off"
+set optionstoggled_text[4]="Load [k]mdb........... On"
 set optionsmenu_command[4]="toggle_kmdb"
-set optionsmenu_keycode[4]=109
-set optionsansi_caption[4]="k^[1mm^[mdb............ ^[34;1mOff^[m"
-set optionstoggled_ansi[4]="k^[1mm^[mdb............ ^[32;7mOn^[m"
-
-set optionsmenu_init[5]="init_debug"
-set optionsmenu_caption[5]="[D]ebug............... Off"
-set optionstoggled_text[5]="[D]ebug............... On"
-set optionsmenu_command[5]="toggle_debug"
+set optionsmenu_keycode[4]=107
+set optionsansi_caption[4]="Load ^[1mk^[mmdb............. ^[34;1mOff^[m"
+set optionstoggled_ansi[4]="Load ^[1mk^[mmdb............. ^[32;7mOn^[m"
+
+set optionsmenu_init[5]="init_drop_into_kmdb"
+set optionsmenu_caption[5]="[D]rop into kmdb...... Off"
+set optionstoggled_text[5]="[D]rop into kmdb...... On"
+set optionsmenu_command[5]="toggle_drop_into_kmdb"
 set optionsmenu_keycode[5]=100
-set optionsansi_caption[5]="^[1mD^[mebug........... ^[34;1mOff^[m"
-set optionstoggled_ansi[5]="^[1mD^[mebug........... ^[32;7mOn^[m"
+set optionsansi_caption[5]="^[1mD^[mrop into kmdb........ ^[34;1mOff^[m"
+set optionstoggled_ansi[5]="^[1mD^[mrop into kmdb........ ^[32;7mOn^[m"
 
 set optionsmenu_init[6]="init_rescue"
 set optionsmenu_caption[6]="[R]escue Mode......... Off"
